base/compiler/typeinfer.jl

```diff
@@ -48,21 +48,22 @@ end
 _typeinf_identifier(frame::InferenceFrameInfo) = frame
 
 """
-    Core.Compiler.Timing(mi_info, start_time, ...)
+    Core.Compiler.Timing(interpT, mi_info, start_time, ...)
 
 Internal type containing the timing result for running type inference on a single
 MethodInstance.
 """
 struct Timing
+    interpT::Type
```

Comment:
```suggestion
    interpT::DataType
```
`typeof` always returns `DataType`.